docs/api.md

```diff
@@ -330,7 +330,7 @@ The synchronization result object exposes the following properties:
 - `ok`: The boolean status of the synchronization operation; `true` if no unresolved conflicts and no errors were encountered.
 - `lastModified`: The timestamp of the latest known successful synchronization operation (no error and no conflict encountered).
 - `conflicts`: The list of unresolved conflicts encountered during both import and export operations (see *[Resolving conflicts manually](#resolving-conflicts-manually)*);
-- `errors`:    The list of encountered errors, if any.
+- `errors`:    The list of encountered errors, if any. Each error has a `type` property, which can be set to `incoming` or `outgoing` depending on which part of the sync flow it's been caught from;
```

Comment:
Nit: Remove 'set to' from the phrase, because this property is something that as a developer you read, not something you write to.